charts/kafka/README.md

```diff
@@ -1,10 +1,19 @@
 # Apache Kafka Helm Chart
 
+![Version: 1.2.0](https://img.shields.io/badge/Version-1.2.0-informational?style=flat-square) ![AppVersion: 3.7.1](https://img.shields.io/badge/AppVersion-3.7.1-informational?style=flat-square)
```

Comment:
Including something like this allows this third party to gather analytics on visitors to our project.  Generally, I am reluctant to do that.